test/house_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def test_it_exists
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_instance_of House, house
+  end
+
+  def test_it_has_rooms
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_equal [], house.rooms
+  end
+
+  def test_it_can_add_rooms
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+
+     house.add_room(room_2)
+     house.add_room(room_1)
+
+     assert_equal [room_2, room_1], house.rooms
+  end
+
+  def test_a_house_is_above_market_average
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_equal false, house.above_market_average?
+  end
```

Comment:
when a method could return true or false, its important to verify both outcomes!  This method is passing this test, but is not really working in the way we expect.